### ISSUE TITLE
Introduce new status for automatic translations

### ISSUE THREAD

**Issue by ulliholtgrave** [digitalfabrik/integreat-cms#1883]
### Motivation

As we intend to use more and more automated translations (e.g. DeepL, Intento, SummAI), we need a new translation status to differentiate them from human translations. This is also important as our customers are using internal sources to validate the quality of the translations (Lektorat). After they checked the translation they should be able to remove the "automatic translation" status and go back to our normal workflow (public, outdated, missing). However, we need to cover four main cases with our translations and their corresponding status:
- XLIFF-Upload of translations that were created by human translators -> Import should result in status "up-to-date" and should be directly visible in the app (publication status: `PUBLIC`)
- XLIFF-Upload of translations that were created by automatic translation services -> Import should result in status "automatic translation" and should be directly visible in the app (publication status: `PUBLIC`)
- Automatic translation with trustful automatic services within our CMS (e.g. DeepL) -> Import should result in status "automatic translation" and should be directly visible in our app (publication status: `PUBLIC`)
- Automatic translation with doubtful automatic services within our CMS (e.g. SummAI) -> Import should result in status "automatic translation" and shouldn't be visible in our app (publication status: `IN_REVIEW`)

### Proposed Solution

For starters I agreed with the service team to introduce a checkbox for the XLIFF-Import that signalizes that the XLIFF is created by a translation service. The imported translations should therefore have the new "automatic translation" status.

Together with the service team we decided to use this lucide icon for the new status:

![Screenshot 2022-12-01 at 23-12-28 Lucide](https://user-images.githubusercontent.com/16021269/205170158-2aee585b-da91-4826-bd8f-4cec5795938e.png)
— https://lucide.dev/icon/bot

After the translations are added users should be able to verify the translation in the page form and the new status should be up-to-date.

### Additional Context

After the implementation of this status automated translations within the system (as from #1727 ) should also get the new status and there should be an option to define "doubtful" translation services.

**Comment by dkehne** [comment]
Yes it is about the translation status. And it can be used to separate "up-to-date-mt-not-lectured" from "up-to-date-human-or-mt-lectured"

i would say the other thing is the page-status

**Comment by timoludwig** [comment]
> Yes it is about the translation status. And it can be used to separate "up-to-date-mt-not-lectured" from "up-to-date-human-or-mt-lectured"

:+1:

> i would say the other thing is the page-status 

The problem is that one "page" can be e.g. public in German and draft in English, so if the status is not unambiguous for a single page, I would not call it the "page status"...

**Comment by timoludwig** [comment]
Apart from that, this issue is essentially a duplicate of #1724, right? :thinking: 
(With the difference that the old issue stated "orange checkmark" as decision, which kind of contradicts the robot-icon :robot:)

**Comment by osmers** [comment]
@timoludwig I think there is also the difference between how the translation should be shown in the app - SUMM AI in draft, DeepL published. Both with the robot icon :)

**Comment by SteffiStuffel** [comment]
Talked with @osmers and @dkehne: SUMM.AI status should be "IN_REVIEW"

**Comment by timoludwig** [comment]
@SteffiStuffel when implementing this, could you also add the new icon to the [translation coverage report](http://localhost:8000/augsburg/analytics/translation-coverage/)? (see [here](https://github.com/digitalfabrik/integreat-cms/issues/1830#issuecomment-1333718136))
Not sure which color to use for bot translations - maybe purple? Or let your creativity run wild :smile:

**Comment by timoludwig** [comment]
Ok, but now I have another question: How are we going to differentiate between machine translations which are up-to-date and those which are outdated?
This sounds a bit confusing to be, since this might lead to users translating the same content they have already translated again just to make sure it's up-to-date, or on the other hand not updating already existing machine translations because they don't see that they're outdated? :thinking: 
Or am I missing something here? Are just using the "outdated" icon for all outdated translations, no matter if machine translations or not, and the bot icon only for up-to-date machine translations?

**Comment by osmers** [comment]
@timoludwig yes exactly - so only after import do we have the robot icon - to show which sections have not been checked. Once they go into the page, change something and update it, the status should be the green check mark
Not sure how to deal with the case that they checked the page but did not change anything bcs it was not necessary and then click update - still a green mark?

Also, if the icon is a robot and something in the German content is changed, the status should change to outdated. Otherwise there is no way to know which are outdated. But I don't think this will happen too often, since the whole point is, that they can update it automatically. But just in case, I think that should be the behaviour.
